src/StarRatingComponent.jsx

```diff
@@ -93,7 +93,7 @@ class StarRatingComponent extends Component {
         <label
           key={`label_${id}`}
           style={starStyles(i, value)}
-          className="dv-star-rating-star"
+          className={'dv-star-rating-star ' + (value >= i ? 'fullStar' : 'emptyStar')}
```

Comment:
@dhbalaji I like this change but could you please make the class names inline with other ones:

- `fullStar` -> `dv-star-rating-full-star`
- `emptyStar` -> `dv-star-rating-empty-star` 

Just FYI there are props for styling color based on the state:

```jsx
    starColor={String} /* color of selected icons, default `#ffb400` */
    emptyStarColor={String} /* color of non-selected icons, default `#333` */
```